### PR TITLE
add 5 missing multiplication symbols

### PR DESCRIPTION
**Pull request by ZacLN**
I've taken the names from http://graphemica.com/

### CODE REVIEW

File: src/token_kinds.jl [hunk 1]
```diff
@@ -1289,6 +1294,11 @@ const UNICODE_OPS = Dict{Char, Kind}(
 '⩠' => LOGICAL_AND_WITH_DOUBLE_UNDERBAR,
 '⫛' => TRANSVERSAL_INTERSECTION,
 '⊍' => MULTISET_MULTIPLICATION,
+'▷' =>WHITE_RIGHT_POINTING_TRIANGLE,
```
Comment (KristofferC) [inline src/token_kinds.jl]: maybe fix the missing space

**Comment by KristofferC** [comment]
How about `WHITE_LEFT_POINTING_TRIANGLE`?

**Comment by ZacLN** [comment]
It's not in `julia-parser.scm`

**Comment by KristofferC** [comment]
Ok

**Comment by KristofferC** [comment]
Merge this and tag so you can use this in v0.7?

**Comment by ZacLN** [comment]
Lets merge but maybe wait until the end of the week to tag?

**Comment by KristofferC** [comment]
Yeah

**Pull request merged**